.github/workflows/ansible-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ansible-version: [stable-2.12]
```

Comment:
Shall we change to 2.14 as minimum as 2.13 is EOL in next month?
or we'll take this change on next release occurrence!

Reply:
will make this change in next month release